conjur/ssl_service.py

```diff
@@ -7,10 +7,17 @@
 """
 # Third party
 import logging
+import os
 import socket
 from OpenSSL import SSL
 from OpenSSL.crypto import FILETYPE_PEM, dump_certificate
 
+_tls_methods = {
+    "1.0": SSL.TLSv1_METHOD,
+    "1.1": SSL.TLSv1_1_METHOD,
+    "1.2": SSL.TLSv1_2_METHOD
```

Comment:
I know TLSv1_3_METHOD is not exist but the user can still specify working with 1.3 only no? If I understand the SSL docs correctly

Reply:
correct TLSv1_3_METHOD doesnt exist but SSLv23_METHOD exists which allows negotiation with 1.3. We can say that if the user specifies 1.3, then 1.2 will also be blocked (`ctx.set_options(SSL.OP_NO_TLSv1_2)`

Something like 
if os.getenv('TLS_VERSION') == "1.3":
  ```
          ctx.set_options(SSL.OP_NO_SSLv2)
          ctx.set_options(SSL.OP_NO_SSLv3)
          ctx.set_options(SSL.OP_NO_TLSv1)
          ctx.set_options(SSL.OP_NO_TLSv1_1)
          ctx.set_options(SSL.OP_NO_TLSv1_2)
  ```